tests/test_commit.py

```diff
@@ -10,6 +10,80 @@
 from git_wrapper import exceptions
 
 
+def test_describe_tag_and_patch(mock_repo):
+    """
+    GIVEN GitRepo is initialized with a path and repo
+    WHEN commit.describe is called with a tag and patch value
+    THEN a dictionary with tag and patch keys is returned
+    """
+    expected = {'tag': '1.0.0', 'patch': '12345'}
+    attrs = {'describe.return_value': '1.0.0-g12345'}
+    mock_repo.git.configure_mock(**attrs)
+
+    repo = GitRepo('./', mock_repo)
+
+    assert expected == repo.commit.describe('12345')
+
+
+def test_describe_tag_only(mock_repo):
+    """
+    GIVEN GitRepo initialized with a path and repo
+    WHEN commit.describe is called a tag value only
+    THEN a dictionary with a tag and empty patch is returned
+    """
+    expected = {'tag': '1.0.0', 'patch': ''}
+    attrs = {'describe.return_value': '1.0.0'}
+    mock_repo.git.configure_mock(**attrs)
+
+    repo = GitRepo('./', mock_repo)
+
+    assert expected == repo.commit.describe('12345')
```

Comment:
should this be 1.0.0?

Reply:
Ok, we can leave it - I was reviewing everything as new, and this one didn't seem right when I read it.